LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2020-2021, Caesar authors & contributors
```

Comment:
```suggestion
Copyright 2020-2021, Caesar authors & contributors
```
the (c) is meaningless, just copyright is sufficient

Reply:
Just copy-pasted the github-supplied template :shrug: